pkg/kev/envs.go

```diff
@@ -69,12 +69,13 @@ func (e *Environment) GetServices() Services {
 
 // GetService retrieves the specific service by name from the environment's overlay.
 func (e *Environment) GetService(name string) (ServiceConfig, error) {
-	for _, s := range e.GetServices() {
-		if s.Name == name {
-			return s, nil
-		}
-	}
-	return ServiceConfig{}, fmt.Errorf("no such service: %s", name)
+	return e.overlay.getService(name)
```

Comment:
a small suggestion. perhaps we should use `override` instead of `overlay` to be more explicit as to what it is? `overlay` is a bit ambiguous. Obviously same comment apply to all instances below.

Reply:
Actually this matches docker-composes [own docs](https://docs.docker.com/compose/extends/).